src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1204,6 +1208,17 @@ static Library()
                 BuiltinFunctionsCore.Pi,
                 NoErrorHandling(Pi)
             },
+            {
+                BuiltinFunctionsCore.PlainText,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.PlainText.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
```

Comment:
>NoOpAlreadyHandledByIR [](http://example.com/codeflow?start=40&length=22)

If we say that it's handled by the IR, we probably need to override GetArgPreprocessor in the function (or make it inherit from StringOneArgFunction instead of BuiltinFunction)

Reply:
Ah I see, I updated this to replace empty string for blank value,

Reply:
I agree with Carlos, we should inherit `StringOneArgFunction` and have this as `NoOpAlreadyHandledByIR` that way it can be handled at IR phase and the advantage is there other backend in future would not need to worry about the blank handling.

Reply:
Updated to use StringOneArgFunction.